tests/utils/nitpick.js

```diff
@@ -281,7 +291,53 @@ setUpTest = function(testCase) {
     // This is needed to enable valid tests when Interface does not have focus
     // The problem is that models aren't rendered when there is no focus
     previousThrottleFPS = Menu.isOptionChecked("Throttle FPS If Not Focus");
-    Menu.setIsOptionChecked("Throttle FPS If Not Focus", false)
+    Menu.setIsOptionChecked("Throttle FPS If Not Focus", false);
+    
+    //Setup variables for Client Profile capabilities
+    graphicsCardType = PlatformInfo.getGraphicsCardType().toLowerCase();
+    
+    graphicsCardVendor = "Unknown";
+    if (graphicsCardType.search("nvidia") !== -1) {
+        graphicsCardVendor = "nvidia";
+    } else if (graphicsCardType.search("radeon") !== -1) {
+        graphicsCardVendor = "radeon";
+    }
+
+    // Extract the graphics card model from the type
+    // This uses a regex, and assumes the model is the first integer following a space
+    // Examples: "NVIDIA GeForce GTX 1070 with Max-Q Design"  => " 1070"
+    //           "Radeon Pro 560"                             => " 560"
+    var regex = / [0-9]+/;
+    graphicsCardModelNumber = parseInt(graphicsCardType.match(regex)); // The parseInt command safely ignores the initial blank
+
+    CPUBrand = PlatformInfo.getCPUBrand();
+    operatingSystemType = PlatformInfo.getOperatingSystemType();
+    isHMDInUse = (PlatformInfo.hasRiftControllers() || PlatformInfo.hasViveControllers());
```

Comment:
Is there a better way to detect this?

Reply:
There is a `HMD.isHMDAvailable()` command - but that returns true if you have a Rift attached, and in Desktop mode.  Querying the availability of controllers verifies that you actually are in VR.

I am not sure yet how tests will work in VR, and this may be updated when I figure that out (i.e. maybe we'll use the Rift Simulator option).